### PR TITLE
feat(): add contact page and add menu icons

### PR DESCRIPTION
**Pull request by thielpa**
add conatact page and add menu icons

Screenshots (ignore the green bar....colors will be changed in a following pr):
image
image
image

### CODE REVIEW

**Comment by thielpa** [comment]
@DevVersion resolved conflicts

**Comment by thielpa** [comment]
@DevVersion added a screenshot without bold username

**Comment by pgschwendtner** [comment]
Thanks for your reply, we will analyze your comment as soon as possible.

File: app/services/SessionAccessor.ts [hunk 1]
```diff
@@ -32,4 +32,22 @@ export class SessionAccessor {
...
+
+  setUsername(username: string) {
+    if (!username) {
+      this.localStorage.remove('VP_USERNAME');
+      this.sqlStorage.remove('VP_USERNAME');
+      return;
+    }
+
+    this.localStorage.set('VP_USERNAME', username);
+    this.sqlStorage.set('VP_USERNAME', username);
```
Comment (devversion) [inline app/services/SessionAccessor.ts]: This is weird. The username is included inside of the token.

Take a look at `jwt-token` to see how you can decode a token.

File: config.xml [hunk 2]
```diff
@@ -5,6 +5,7 @@
   <author email="paulgschwendtner@me.com" href="http://www.github.com/DevVersion/">Paul Gschwendtner</author>
   <content src="index.html"/>
   <access origin="*"/>
+  <access origin="mailto:*" launch-external="true"/>
```
Comment (devversion) [inline config.xml:8]: :+1:

File: app/pages/representation/representation.ts [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import {Page, NavController, Modal, MenuController} from "ionic-angular/index";
+import {Page, NavParams, NavController, Modal, MenuController} from "ionic-angular/index";
```
Comment (devversion) [inline app/pages/representation/representation.ts]: Unnecessary import.

File: app/pages/representation/representation.html [hunk 4]
```diff
@@ -1,17 +1,34 @@
 <ion-menu [content]="content">
-  <ion-toolbar primary>
+  <ion-toolbar secondary>
```
Comment (DevVersion) [inline app/pages/representation/representation.html]: I don't know, but there is a really ugly looking border around the username?

File: app/pages/login/login.ts [hunk 5]
```diff
@@ -25,6 +25,7 @@ export class LoginPage {
     this.backend.sendLoginRequest(username, password).then((payload) => {
       this._switchToRepresentationPage();
       this.session.setToken(payload['token']);
+      this.session.setUsername(username);
```
Comment (devversion) [inline app/pages/login/login.ts]: As mentioned below, you should use the token decoding.

File: app/pages/representation/representation.html (hunk 4 above)
Comment (Lordnoname) [inline app/pages/representation/representation.html]: Already fixed

File: app/pages/contact/contact.html [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+        <span padding-left>
+          <strong>{{ currentUser }}</strong>
+        </span>
+    </div>
+  </ion-toolbar>
+  <ion-content>
+    <ion-list>
+      <button ion-item (click)="showRepresentations()">
+        <ion-icon name="calendar"></ion-icon>
+        <span padding-left>{{ currentClass !== '*' ? 'Klasse ' + currentClass : 'Klassen' }}</span>
```
Comment (devversion) [inline app/pages/contact/contact.html:17]: :+1:

**Pull request merged**